### PR TITLE
refresh token

### PR DESCRIPTION
**Pull request by Eleonore9**
* new table in cassandra `refresh_tokens`
* create a token pair

### CODE REVIEW

File: src/kixi/heimdall/components/database.clj [hunk 1]
```diff
@@ -82,6 +85,12 @@
       (if (coll? result)
         (map (partial into {}) reformatted)
         reformatted)))
+  (update! [this table what where]
+    (exec this (hayt/update table (hayt/set-columns
+                                   (reduce-kv #(assoc %1 (util/hyphen->underscore %2) %3)
```
Comment (acron0) [inline src/kixi/heimdall/components/database.clj]: Not sure you need `reduce-kv` here - the [replacer](https://github.com/MastodonC/kixi.heimdall/blob/dcc6b479a1958dcf4222cc30de8c0376f6363aaa/src/kixi/heimdall/util.clj) should cope with maps.

Comment (elisehuard) [inline src/kixi/heimdall/components/database.clj]: using that returns a vector (we tried first to do like other methods of protocol, doing apply map etc) and this function requires a hashmap as output
unless there's a way to use it we're missing?

Comment (acron0) [inline src/kixi/heimdall/components/database.clj]:
```
(into {} (underscore->hyphen {:a_b 1 :c_d 2}))
```
This will do what you need

**Comment by acron0** [comment]
:shipit:

**Pull request merged**